services/gundeck/src/Gundeck/Env.hs

```diff
@@ -77,7 +77,7 @@ createEnv m o = do
             . C.setResponseTimeout 10
             . C.setProtocolVersion C.V3
             $ C.defSettings
-    a <- Aws.mkEnv l o n
```

Comment:
Then `n` probably shouldn't be created earlier?  Or, it if is used somewhere else, we should use the one created in `mkEnv`?